### PR TITLE
Updated pyscf_to_trexio.py

### PR DESCRIPTION
**Pull request by kousuke-nakano**
Hi. I have upadted pyscf_to_trexio.py. It is splitted to two files.
1. pyscf_example.py -> generating a pyscf chk file.
2. pyscf_to_trexio.py -> converting a pyscf chk file to a TREXIO file.
The converter supports both molecules and crystals(PBC).
Several examples files are also added.

### CODE REVIEW

File: src/converters/pyscf_to_trexio/pyscf_example.py [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+stream_handler.setLevel(log_level)
+#handler_format = Formatter('%(name)s - %(levelname)s - %(lineno)d - %(message)s')
+handler_format = Formatter('%(message)s')
+stream_handler.setFormatter(handler_format)
+logger.addHandler(stream_handler)
+
+# load python packages
+import numpy as np
+import os, sys
+import pandas as pd
```
Comment (q-posev) [inline src/converters/pyscf_to_trexio/pyscf_example.py:29]: Do not forget to apply suggestions from my previous code review (e.g. remove unnecessary imports).

**Comment by q-posev** [comment]
We have a dedicated `data` directory in this repository. Could you move the data files there and modify the paths accordingly? Thanks!

File: src/converters/pyscf_to_trexio/pyscf_example.py (hunk 1 above)
Comment (kousuke-nakano) [reply src/converters/pyscf_to_trexio/pyscf_example.py:29]: Sorry, I have removed them only from pyscf_to_trexio.py. I will do this.

**Comment by q-posev** [comment]
Hi @kousuke-nakano! Feel free to push your latest changes, i can finish the rest.

**Comment by kousuke-nakano** [comment]
Hi @q-posev, I have pushed my latest changes. One can convert a pyscf chkfile by:

`python pyscf_to_trexio.py -c diamond_single_k.chk -o diamond_single_k.hdf5`

Let me discuss how to merge "pyscf_to_trexio.py". "pyscf_to_trexio.py" works as a stand-alone tool. Do you think we have to integrate it into "convert_to.py"? or I can leave it a stand-alone tool, and put a new line in setup.cfg?

**Comment by q-posev** [comment]
Great work @kousuke-nakano, thanks ! I am merging it as is and will do the necessary rearrangements in a separate PR, will put you as a reviewer.

**Comment by kousuke-nakano** [comment]
Hi @q-posev, Sure! Thanks.

**Pull request merged**